standalone_self_upgrade.cf.in

```diff
@@ -142,6 +142,14 @@ bundle agent cfengine_software
         comment => "On debian hosts it's the standard to use 'amd64' instead of
                    'x86_64' in package architectures.";
 
+    (debian|ubuntu).aarch64::
+
+      "pkg_arch"
+        string => "arm64",
+        comment => "On debian hosts it's the standard to use 'arm64' instead of
+                   in package architectures.";
```

Comment:
not sure I understand this sentence? Maybe

```
On debian hosts it's the standard to use 'arm64' instead of the package architectures.
```

?

https://wiki.debian.org/ArchitectureSpecificsMemo

>Architecture baselines
>Each Debian architecture has a baseline indicating the oldest or least capable CPU on which the architecture can be used. The baseline can change between Debian releases. The baseline is mostly defined by the gcc-N package, which is configured to produce baseline binaries when options like -march= are not used.

and later

>arm64
>aarch64 (ARMv8-A) with no optional extensions. VFPv4 and NEON are OK, but ARMv8.1-A and up are not guaranteed.

Reply:
I guess this is more related to our package names having arm64 in the file name.